src/Monitor/Monitor/Alerts/NewAzureRmMetricAlertRuleV2CriteriaCommand.cs

```diff
@@ -47,36 +48,47 @@ public class NewAzureRmMetricAlertRuleV2CriteriaCommand : MonitorCmdletBase
         [Parameter(ParameterSetName = DynamicThresholdParameterSet, Mandatory = true, HelpMessage = "Switch parameter for using Dynamic Threshold Type")]
         public SwitchParameter DynamicThreshold { get; set; }
 
+        /// <summary>
+        /// Gets or sets the rule AvailabilityCriteria
+        /// </summary>
+        [Parameter(ParameterSetName = AvailabilityParameterSet, Mandatory = true, HelpMessage = "Switch parameter for using availability criteria Type")]
+        public SwitchParameter WebTest { get; set; }
```

Comment:
I think `-WebTestId` can already differentiate the parameter set, so `-WebTest` can be removed.

Reply:
> I think `-WebTestId` can already differentiate the parameter set, so `-WebTest` can be removed.

@isra-fel Thank you for the review,
I consulted about it with our PM @harelbr and he asked to keep this parameter and make it optional.